packages/autocomplete-plugin-tags/package.json

```diff
@@ -25,9 +27,17 @@
     "build:esm": "babel src --root-mode upward --extensions '.ts,.tsx' --out-dir dist/esm --ignore '**/*/__tests__/'",
     "build:types": "tsc -p ./tsconfig.declaration.json --outDir ./dist/esm",
     "build:umd": "rollup --config",
-    "build": "rm -rf ./dist && yarn build:umd && yarn build:esm && yarn build:types",
-    "on:change": "concurrently \"yarn build:esm\" \"yarn build:types\"",
-    "prepare": "yarn run build:esm && yarn build:types",
+    "build:css": "yarn build:css:minified && yarn build:css:unminified",
+    "build:css:minified": "MINIFIED=TRUE node ../../scripts/buildCss.mjs src/theme.scss dist/theme.min.css",
+    "build:css:unminified": "node ../../scripts/buildCss.mjs src/theme.scss dist/theme.css",
+    "build": "rm -rf ./dist && yarn build:umd && yarn build:esm && yarn build:types && yarn build:css",
+    "on:change": "concurrently \"yarn build:esm\" \"yarn build:types\" \"yarn build:css\"",
+    "prepare": "yarn run build:esm && yarn build:types && yarn build:css",
     "watch": "watch \"yarn on:change\" --ignoreDirectoryPattern \"/dist/\""
+  },
+  "devDependencies": {
+    "@algolia/autocomplete-core": "1.2.2",
+    "@algolia/autocomplete-js": "1.2.2",
+    "@algolia/autocomplete-shared": "1.2.2"
```

Comment:
We'll need to update the versions before merging.

Reply:
Shouldn't Ship.js do it for us?

Reply:
The version in the `next` branch is 1.3.0 (the base branch got outdated), so the package will not be considered as part of the monorepo, and fetch the Autocomplete version from npm.

Reply:
Addressed in dc7528b.